packages/headless/src/database/mongodb/user-auth/user.model.ts

```diff
@@ -0,0 +1,62 @@
+import { model, Schema } from 'mongoose';
+import { UserEntity } from 'src/entity/user';
+
+/**
+ * User Schema
+ */
+const UserSchema = new Schema<UserEntity>({
+  firstName: {
+    type: String,
+    trim: true,
+    default: '',
+    required: true,
```

Comment:
remove these flag from db level. so that for any change we dont have to change in all db implementations. our service layer will be responsible for checking what is required and what is not

Reply:
ok